### DISCUSSION TITLE
Deactivating the Comments

### DISCUSSION THREAD

**Discussion by SHogenboom** [Erudika/scoold#401] [Q&A]
Is it possible to deactivate comments? I find we lose quite a lot of useful information because the comments are not included in the search. Of course the alternative would be to include comments in the search ;) But for user friendliness I prefer to have only answers.

**Comment by albogdano** [comment]
You can set `scoold.new_users_can_comment = false` but that will onot disable commenting completely. Also coments should appear in search results. What is the problem there?
